src/javascript/app/base/header.js

```diff
@@ -298,11 +298,11 @@ const Header = (() => {
 
                 switch (string) {
                     case 'unsubmitted': {
-                        result = verification_length === 2 && identity.status === 'none' && document.status === 'none';
+                        result = verification_length === 3 && identity.status === 'none' && document.status === 'none' && income.status === 'none';
```

Comment:
for me this statement verification_length === 3 looks quite confusing. what is the possibility for this variable to be truthy?
because in verification array we can have only 'income' item, so verification_length === 3 will return false
maybe we better should check that this array exists at all...
maybe previously it was reasonable, because identity and document are requested together?

Reply:
hope you understand my thought

Reply:
Yes @yauheni-kryzhyk-deriv , but this is just 1 of the message banner asking user to submit their Proof of income, Proof of identity & Proof of address